### ISSUE TITLE
Missing image metadata

### ISSUE THREAD

**Issue by octogonz** [davej/NLogo#4]
Hey @davej I noticed that "**Use svgo to optimise images**" (ff5734ccaf305626dad49b551ec6542be48ef6c9) stripped the copyrights and other metadata from our logos.  Could you restore that?

**Comment by davej** [comment]
Sure, I will do that.

In future, if I continue to use svgo but with `removeComments` set to `false` will that be ok? Or is there other metadata that I should be retaining.

**Comment by octogonz** [comment]
I played around with the `svgo` tool a bit.  This command line produced an output that was substantially smaller, but didn't degrade the quality or remove the copyrights.

```
svgo --disable=removeComments --disable=removeMetadata --disable=cleanupNumericValues rush.svg
```

Maybe we could go with that?

**Comment by davej** [comment]
Fixed in `v1.7.1`

**Comment by octogonz** [comment]
Thanks!
